docs/devguide/contributing.asciidoc

```diff
@@ -40,6 +40,18 @@ you'd like to share with others, open a PR to add it to our list of
 https://github.com/elastic/beats/blob/master/libbeat/docs/communitybeats.asciidoc[community
 Beats].
 
+
+[float]
+[[adding-new-dashboard]]
+=== Adding a New Dashboard
+
+If you want to create a new Dashboard, please read <<community-dashboards>>. You don't need to
+submit the code to this repository. Most new Beats start in their own repository
```

Comment:
I think you need to modify the rest of this text to apply to creating a new dashboard, not creating new Beats.